client/pages/offer-thread.jsx

```diff
@@ -62,30 +86,33 @@ class OfferThread extends React.Component {
     })
       .then(response => response.json())
       .then(reply => {
-
       })
       .catch(err => console.error(err));
     this.setState({ reply: '' });
   }
 
+  componentDidUpdate() {
+    this.scrollToBottom();
+  }
+
+  scrollToBottom() {
+    this.messagesEnd && this.messagesEnd.scrollIntoView({ behavior: 'smooth' });
+  }
```

Comment:
we don't need to do this here if you write the componentDidUpdate that i mentioned before.

 https://reactjs.org/docs/react-component.html#componentdidupdate